sample/src/main/res/layout/item_down_to_earth.xml

```diff
@@ -32,7 +32,6 @@
         android:textSize="15sp"
         android:textStyle="normal"
         app:readLessText="@string/read_less"
-        app:readMoreText="@string/read_more"
```

Comment:
It was added incorrectly in [this commit](https://github.com/webtoon/ReadMoreTextView/commit/ff2ee5b570aeb236fe911ba2a0af699077315cae#diff-0364d6e9ca555fad848c794f032f0f868c48d489671f14931e733fbbf8fdbcb2R35).